verl/workers/actor/dp_actor.py

```diff
@@ -516,7 +520,12 @@ def update_policy(self, data: DataProto):
 
                     policy_loss = pg_loss
                     if calculate_entropy and entropy is not None:
-                        entropy_agg = agg_loss(loss_mat=entropy, loss_mask=response_mask, loss_agg_mode=loss_agg_mode)
+                        entropy_agg = agg_loss(
+                            loss_mat=entropy,
+                            loss_mask=response_mask,
+                            loss_agg_mode=loss_agg_mode,
+                            **self.config.global_batch_info,
```

Comment:
I think `dp_size`, `global_batch_size` and `batch_num_tokens` is only needed in new model engine backend, legacy backend should not pass these parameters. cc @vermouth1992